recipes/pyinterval/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pyinterval" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 8c46224a05815affa803ed5620432236bba620a246701b860d98ae7358a41d21
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - crlibm
+    - six
+
+test:
+  imports:
+    - interval
+
+about:
+  home: "https://github.com/taschini/pyinterval"
+  license: BSD
```

Comment:
Which BSD licence is it? The SPDX identifiers are normally of the form `BSD-N-Clause` where N is a number

Reply:
It looks like 3 clause, but IANAL. I've changed that. Is the family fine?

Reply:
Yup :+1: